src/components/fields/text-field/text-field.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import has from 'lodash.has';
-import uuid from 'uuid';
```

Comment:
Did we remove it from the pkg.json too? I can't see that change.

Reply:
It wasn't there in the first place 🤷‍♂️

Reply:
Then I'd recommend using `depcheck` on the package once. It's awesome and is how I managed to clean up the deps on the mc-fe without much manual work.